### PR TITLE
File Mover

### LINKED ISSUE #93: Script to Organize Files in Your System.
# Aim

- This script should automatically organizes your complete system making. You can use modules such as os and shutil for this purpose.

# Details

- Movies files of certain extension to specified location
- Segregate files according to their types.

### PR DESCRIPTION
**Pull request by Kreateer**
# Description

This is a small program that moves and sorts files from chosen source to destination. It uses PySimpleGUI to create a minimal GUI for the user.
It allows the user to move/copy and sort their files into separate directories based on file type.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [X ] Yes
- [ ] No

## Type of change

- [X] New feature (non-breaking change which adds functionality

## Checklist:

- [X] My code follows the style guidelines(Clean Code) of this project
- [X] I have performed a self-review of my own code
- [X] I have created a helpful and easy to understand `README.md`
- [X] My changes generate no new warnings

### CODE REVIEW

File: File Mover/fmover.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                    if os.path.exists(str(destination) + '/' + 'Text Files'):
+                        return str(os.path.join(str(destination) + '/' + 'Text Files'))
+                    else:
+                        os.mkdir(str(destination) + '/' + 'Text Files')
+                        return str(os.path.join(str(destination) + '/' + 'Text Files'))
+
+            else:
+                sg.PopupError("File type not found!")
+                raise SystemExit()
+
```
Comment (avinashkranjan) [inline File Mover/fmover.py:220]: Delete these empty lines

File: File Mover/fmover.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+                    return str(os.path.join(str(destination) + '/' + 'Archives'))
+
+        # For text files
+            elif type in textf_list:
+                    if os.path.exists(str(destination) + '/' + 'Text Files'):
+                        return str(os.path.join(str(destination) + '/' + 'Text Files'))
+                    else:
+                        os.mkdir(str(destination) + '/' + 'Text Files')
+                        return str(os.path.join(str(destination) + '/' + 'Text Files'))
+
```
Comment (avinashkranjan) [inline File Mover/fmover.py:216]: Add Some for types like pdf, jpg, word most common ones..

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Rest Looks Great..Works Like a Charm..Just do the mentioned changes..

File: File Mover/fmover.py (hunk 1 above)
Comment (Kreateer) [reply File Mover/fmover.py:235]: @avinashkranjan I've deleted them. Somehow I missed that xD

File: File Mover/fmover.py (hunk 2 above)
Comment (Kreateer) [reply File Mover/fmover.py:231]: @avinashkranjan I've re-made the GUI dropdown option and grouped up file types, so when the user gets to the main window, they will see options to choose to move/copy all Archives, Images or Text Files at once :smile:
I suggest you run it to see what I mean exactly xD

**Comment by Kreateer** [comment]
@avinashkranjan I've made some new changes to the script.

Once run, the dropdown option in the GUI no longer shows specific file types, but type groups instead.
So, if you want to move/copy Archive files (.zip, .rar, .7z...), you just select the 'Archives' option and the program automatically identifies and moves all archive files from source to destination 😄

**Review by avinashkranjan — APPROVED** [review]
Works Great..🚀

**Pull request merged**